### PR TITLE
[CIAPP] Add Codefresh integration documentation

### PR DESCRIPTION
**Pull request by cg14823**
### What does this PR do?

Add instruction to setup the Codefresh datadog integration

### Preview

https://docs-staging.datadoghq.com/carlos.gonzalez/add-codefresh-integration-docs/continuous_integration/setup_pipelines/codefresh/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by juan-fernandez — APPROVED** [review]
LGTM

File: content/en/continuous_integration/setup_pipelines/codefresh.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Set up Tracing for Codefresh pipelines
```
Comment (fermayo) [inline content/en/continuous_integration/setup_pipelines/codefresh.md:2]:
```suggestion
title: Set up Tracing on Codefresh pipelines
```

**Review by fermayo — APPROVED** [review]

**Review by maycmlee — APPROVED** [review]
Hey @cg14823! I made a small edit and committed. Should I go ahead and merge this?

**Comment by cg14823** [comment]
@maycmlee yes please

**Pull request merged**